rdma_link_linux.go

```diff
@@ -143,3 +143,128 @@ func (h *Handle) RdmaLinkSetName(link *RdmaLink, name string) error {
 
 	return execRdmaSetLink(req)
 }
+
+func netnsModeToString(mode uint8) string {
+	netnsModes := map[uint8]string{0: "exclusive", 1: "shared"}
+
+	if mode > (uint8)(len(netnsModes)-1) {
+		return "unknown"
+	}
+	return netnsModes[mode]
+}
+
+func executeOneGetRdmaNetnsMode(data []byte) (string, error) {
+	var modeResult string
```

Comment:
please remove this variable.
`return netnsModeToString(mode), nil` and `return "", fmt.Errorf(...)`